content/doc/book/pipeline/getting-started.adoc

```diff
@@ -326,6 +326,9 @@ etc. Consult the built-in global variable reference at
 `$\{YOUR_JENKINS_URL}/pipeline-syntax/globals`
 for a complete, and up to date, list of properties available on `currentBuild`.
 
+This video reviews the usage of the `currentBuild` variable in Jenkins Pipeline.
```

Comment:
```suggestion
This video reviews using the `currentBuild` variable in Jenkins Pipeline.
```

Reply:
absolutely makes sense for active voicing, thanks so much!